reflekt-plugin/src/main/kotlin/io/reflekt/plugin/generation/ir/ReflektIrGenerationExtension.kt

```diff
@@ -1,16 +1,20 @@
 package io.reflekt.plugin.generation.ir
 
 import io.reflekt.plugin.analysis.models.ReflektContext
+import io.reflekt.plugin.utils.Util.log
```

Comment:
maybe run 'optimize imports' on the whole project?

Reply:
I've done it

Reply:
I've done it